WRITING_PLUGINS.md

```diff
@@ -12,6 +12,15 @@ Farscape.register_plugin(name: :cachinator, type: :cache, ...)
 
 Feel free to put this code at the bottom of `cachinator.rb` so that it turns on automatically after the gem is loaded. Consumers who want control can include a line in their initializer reading `Farscape.disable(:cachinator)` or `Farscape.disable(:cache)`. If you'd rather have your plugin be off by default, you could instead wrap the register_plugin call in, say, a `Cachinator.activate` method for the consumer to call as desired. If your plugin is http-specific (say it adds Authentication headers), include `protocol: :http`.
 
+Consumers may also reendable the plugin by using the `reenable` method, which has the same interface as `register_plugin`, but only works is the plugin is already disabled.
```

Comment:
Typo, "works is" for "works if". Also, hm. Would it be better to just call this `enable` and have it work no matter what? The concept seems fine, probably cleaner than what I'd been picturing.